dashboards/mariadb/mariadb.json

```diff
@@ -0,0 +1,2678 @@
+{
+  "name": "MariaDB Overview",
+  "description": null,
+  "pages": [
+    {
+      "name": "MariaDB Overview",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 3,
+            "height": 4
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![MariaDB logo](https://avatars.githubusercontent.com/u/5877084?s=280&v=4)"
```

Comment:
Hi @rahulbasu-nr , to be consistent, can we use this image URL ?

```
https://raw.githubusercontent.com/newrelic/newrelic-quickstarts/bf4f357f1b0eb51e7edd3d617df9370ae5ea6189/quickstarts/mariadb/logo.svg
```

Reply:
adjusted the image URL